src/writing-policies/spec/03-validating-policies.md

```diff
@@ -0,0 +1,71 @@
+# Validating policies
+
+The Kubewarden policy server receives
+[`AdmissionReview`](https://godoc.org/k8s.io/api/admission/v1#AdmissionReview)
+objects from the Kubernetes API server. It then forwards the value of
+the `request` (of type
+[`AdmissionRequest`](https://godoc.org/k8s.io/api/admission/v1#AdmissionRequest)
+key to the policy to be evaluated.
+
+The policy has to evaluate the `request` and state whether it should be
+accepted or not. When the request is rejected, the policy must provide the
+explanation message that is going to be shown to the end user.
+
+By convention of the `policy-server` project, the guest has to expose
+a function named `validate`, exposed through the waPC guest SDK, so
+that the `policy-server` (waPC host) can invoke it.
+
+The `validate` function receives a `ValidationRequest` object serialized as JSON and
+returns a `ValidationResponse` object serialized as JSON.
```

Comment:
Likewise, I see the value in naming it so we describe it later on the next section, but seems a little weird to me to name an unnamed JSON struct.

Reply:
Resolved with a new commit